docs/how-tos/howto-stake-unstake.md

```diff
@@ -0,0 +1,130 @@
+---
+id: howto-stake-unstake
+title: How to set up and use Proof of Stake (PoS)
+---
+
+## Overview
+
+This guide goes into detail on how to set up a Proof of Stake network with the Polygon SDK, how to stake funds for 
+nodes to become validators and how to unstake funds.
+
+It **highly encouraged** to read and go through the [setup IBFT locally](/docs//how-tos/howto-setup-ibft/howto-set-ibft-locally)
+/ [setup IBFT on the cloud](/docs//how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud) sections, before going along with this PoS guide. 
+These sections outline the steps needed to start a Proof of Authority (PoA) cluster with the Polygon SDK.
+
+:::warning PoA and PoS chains incompatible
+A chain that has previously been running in PoA mode, cannot be converted to a PoS chain during execution, and vice versa.
+
+The only way to switch IBFT mechanisms is to reset the chain and start it in the corresponding mode (PoA or PoS) from scratch.
+:::
+
+Currently, there is no limit to the number of validators that can stake funds on the Staking Smart Contract.
+
+## Staking Smart Contract
+
+The repo for the Staking Smart Contract is located [here](https://github.com/0xPolygon/staking-contracts).
+
+It holds the necessary testing scripts, ABI files and most importantly the Staking Smart Contract itself.
+
+## Setting up an N node cluster
+
+Setting up a cluster with the Polygon SDK is covered in the [setup IBFT locally](/docs/how-tos/howto-setup-ibft/howto-set-ibft-locally)
+/ [setup IBFT on the cloud](/docs/how-tos/howto-setup-ibft/howto-set-ibft-on-the-cloud) sections.
+
+The **only difference** between setting up a PoS and PoA cluster is in the genesis generation part.
+
+**When generating the genesis file for a PoS cluster, an additional flag is needed `--pos`**:
+```bash
+go run main.go genesis --pos ...
```

Comment:
In my opinion, i think it would be better to have the whole command. That way user can use the copy feature of the field and not go to another page

Reply:
There is no need to add additional flags and write out full commands if the doc is supposed to just highlight changes between two cluster initializations.

The correct way to initialize a genesis file is already covered extensively in the 2 guides referred back to in the article multiple times